projects/commudle-admin/src/app/feature-modules/external-feed/components/external-feed/external-feed.component.scss

```diff
@@ -1,21 +1,71 @@
 @import '/projects/commudle-admin/src/assets/styles/variables';
+
 .external-feed {
-    width: 100%;
-  
+  .left-column {
+    width: $small-column-width;
+
     .header {
+      border: $border-1 solid $solitude;
+      border-radius: $border-16;
+      box-shadow: none;
+      height: 60px;
+      font-weight: $fw-500;
+      font-size: $fs-25;
+      text-align: justify;
+    }
+
+    .tag-checkboxes {
+      border: $border-1;
+      border-radius: $border-16;
+      box-shadow: none;
+
+      font-weight: $fs-h7;
+      font-size: $fs-30;
+      text-align: left;
+
+      margin-top: $space-4;
+      margin-left: $space-10;
+
+      padding-left: $space-32;
+      padding-top: $space-20;
+      padding-bottom: $space-20;
+
+      .tag-checkbox {
+        font-size: $fs-16;
+
+        .label {
+          display: inline-block;
+        }
+      }
+    }
+  }
+
+  .right-column {
+    width: $media-width;
```

Comment:
please match left and right column width with this page as discussed: https://commudle.com/labs/intro-to-data-structures-in-c-basic-level

Reply:
Done for left column